osu.Game/Screens/Play/Player.cs

```diff
@@ -143,8 +144,12 @@ private void load(AudioManager audio, APIAccess api, OsuConfigManager config)
 
             adjustableClock.ProcessFrame();
 
+            // Lazer's audio timings in general doesn't match stable. This is the result of user testing, albeit limited.
+            // This only seems to be required on windows. We need to eventually figure out why, with a bit of luck.
+            var offsetClock = new FramedOffsetClock(adjustableClock) { Offset = RuntimeInfo.OS == RuntimeInfo.Platform.Windows ? 22 : 0 };
+
             // the final usable gameplay clock with user-set offsets applied.
-            var offsetClock = new FramedOffsetClock(adjustableClock);
+            offsetClock = new FramedOffsetClock(offsetClock);
 
             userAudioOffset.ValueChanged += v => offsetClock.Offset = v;
```

Comment:
this is going to override the offset set above

Reply:
There are two offset clocks.

Reply:
check kline above